scripts/xtensa-build-all.sh

```diff
@@ -30,30 +39,56 @@ usage: xtensa-build.sh [options] platform(s)
        -u Force UP ARCH
        -d Enable debug build
        -c Interactive menuconfig
+       -o copies the file argument from src/arch/xtensa/configs/override/$arg.config
+	  to the build directory after invoking CMake and before Make.
        -k Use private key
        -v Verbose Makefile log
        -j n Set number of make build jobs. Jobs=#cores when no flag. \
 Infinte when not specified.
+	-m path to MEU tool. Switches signing step to use MEU instead of rimage.
+	-s path to openssl tool. Requires -m and required by -m.
```

Comment:
If required by m then I would mention this in m

Reply:
I'm recommending we remove this option below, simple pass-through instead.